Scripts/DCS-BIOS/doc/addresses.txt

```diff
@@ -47,6 +47,7 @@ FC3				0x6000	#4
 AH-6J				0x6200	#7		(Mod)
 Alphajet			0x6400	#9		(Mod)
 F-22A				0x6600	#40		(Mod)
+MH-60R				0xA400	#47

```

Comment:
These should be in order by address I believe

Reply:
There's already a line for 0xa400 so just add it there

Reply:
yes and those #id should be removed, they are for DCSFP but it has it's own file now.

Reply:
Just check with @WarLord211 on this one, he typically maintains this file. I don't use it personally.

Reply:
I think the ID in the bios.lua comment for the line is supposed to match the # here as well (again, not sure why)

Reply:
yes but they are not necessary anymore, DCSFP don't read any module info from DCS-BIOS anymore. Manually entered into a file. User can do that if a module is missing until it is added in the release.